mainapp/views.py

```diff
@@ -44,6 +51,144 @@ class CreateRequest(CreateView):
     success_url = '/req_sucess/'
 
 
+
+    def form_valid(self, form):
+
+        self.object = form.save()
+
+        oauth_url = 'http://35.200.150.137/platform/oauth/token'
+
+        oauth_d = {
+        "grant_type": "password",
+        "scope": "apikeys posts country_codes media forms api tags savedsearches sets users stats layers config messages notifications webhooks contacts roles permissions csv tos dataproviders",
+        "client_id": "ushahidiui",
+        "client_secret": "35e7f0bca957836d05ca0492211b0ac707671261",
+        "api_key" : "db94a1a0-f89e-4035-8003-fd16a0a98ef5",
+        "password": "keralarescue", 
```

Comment:
can we please get this from an environment variable going through `settings` ? AND add a setting like `POST_TO_USHAHIDI` as `True` or `False` - the big problem with this current setup is that when people are developing / testing locally, it will still push to Ushahidi, which is not good.

Reply:
we just wanna see if the setup works, if it works we'll shift critical data